dashboard/config/linux/bits/arm_emu.yml

```diff
@@ -0,0 +1,10 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+config:
+ # We reduce number of vivid/netrom/rose/hcd devices to 2 because we use at most 2 procs
+ # with emulation and creating these devices takes time during boot.
+ - CMDLINE: [append, "smp.csd_lock_timeout=300000 watchdog_thresh=165 workqueue.watchdog_thresh=420 sysctl.net.core.netdev_unregister_timeout_secs=420 dummy_hcd.num=2"]
```

Comment:
I think this is not really about arm. x86 and riscv use the same numbers based on native/emulation.
It makes sense to remove arm from the name (maybe add "timeouts" to the name since this is all about tuning timeouts) and reuse for x86/riscv as well.

Reply:
Made it more generic. PTAL.